pkg/adapters/http.go

```diff
@@ -245,6 +245,7 @@ func (c *HTTPClient) CreatePullRequestFromTemplate(params capi.CreatePullRequest
 
 	res, err := c.client.R().
 		SetHeader("Accept", "application/json").
+		SetHeader("Git-Provider-Token", params.GitProviderToken).
```

Comment:
Should this reference a `const` that was set somewhere? I think there is a `GitProviderTokenHeaderName` or something...somewhere.